src/components/App/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import Header from '../Header';
+import HomePage from '../HomePage';
+import MapPage from '../MapPage';
+import AllCentersPage from '../AllCentersPage';
+// import MoreInfoPage from '../MoreInfoPage';
+
+const App = () => {
+  return (
+    <>
+      <Header />
+      <Switch>
```

Comment:
Good job on the pathing nice job on the exact vs non exact paths excited to try them out.